test/e2e/tests/releases/utils/methods.ts

```diff
@@ -0,0 +1,211 @@
+import {type SanityClient, type SanityDocument} from '@sanity/client'
+
+export const CLIENT_OPTIONS = {
+  apiVersion: 'v2025-02-19',
+}
+
+export const createRelease = async ({
+  sanityClient,
+  dataset,
+  releaseId,
+  metadata,
+}: {
+  sanityClient: SanityClient
+  dataset: string | undefined
+  releaseId: string
+  metadata: Record<string, any>
+}) => {
+  await sanityClient.withConfig(CLIENT_OPTIONS).request({
+    uri: `/data/actions/${dataset}`,
+    method: 'POST',
+    body: {
+      actions: [
+        {
+          actionType: 'sanity.action.release.create',
+          releaseId: releaseId,
+          metadata: metadata,
+        },
+      ],
+    },
+  })
+}
+
+export const archiveRelease = async ({
+  sanityClient,
+  dataset,
+  releaseId,
+}: {
+  sanityClient: SanityClient
+  dataset: string | undefined
+  releaseId: string
+}) => {
+  const query = `*[_type == "system.release" && _id == "_.releases.${releaseId}"][0]`
```

Comment:
```suggestion
  const query = `*[_type == "system.release" && _id == "_.releases.${releaseId}"][0] {state}`
```
(if we really need to query for state first)